### ISSUE TITLE
`comma-dangle` false positive when a method argument is a string splitted on multiple lines

### ISSUE THREAD

**Issue by plbrault** [eslint/eslint#7571]
**Tell us about your environment**

* **ESLint Version:** 3.9.1
* **Node Version:** 6.9.1
* **npm Version:** 3.10.8

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

```
module.exports = {
  extends: 'airbnb',
  plugins: [
  ],
  rules: {
    'no-console': 'off'
  }
};
```

Configuration Dependencies:

 "eslint-config-airbnb": "^13.0.0",
 "eslint-plugin-import": "^2.2.0",
 "eslint-plugin-jsx-a11y": "^2.2.3",
 "eslint-plugin-react": "^6.6.0"

**What did you do? Please include the actual source code causing the issue.**

```js
  console.log(
    `--- START ---\n`
    + `Received new request at ${(new Date()).toISOString()}\n`
    + `API Key: ${req.query.api_key}\n`
    + `Topic: ${req.query.topic}\n\n`
    + `${req.body}\n`
    + '--- END ---'
  );
```

**What did you expect to happen?**

There should be no linting errors or warnings on this.

**What actually happened? Please include the actual, raw output from ESLint.**

The rule `comma-dangle` fails on the last line of the string, as if it was an object.

```
  41:20  error  Missing trailing comma  comma-dangle

✖ 1 problem (1 error, 0 warnings)
```

**Comment by not-an-aardvark** [comment]
Thanks for the report. Would you mind running `eslint --print-config yourFile.js` on your file and pasting the output (particularly for the `comma-dangle` rule)? This might make it easier to figure out what part of the rule is causing the problem.

**Comment by plbrault** [comment]
It looks like it is coming from Airbnb's config (the `"functions": "always-multiline"` part), so it may not be an ESLint issue after all.

```
    "comma-dangle": [
      "error",
      {
        "arrays": "always-multiline",
        "objects": "always-multiline",
        "imports": "always-multiline",
        "exports": "always-multiline",
        "functions": "always-multiline"
      }
    ],
```

However, it seems like it is a syntax error to put a trailing comma at the end of a function argument list.

**Comment by ljharb** [comment]
@plbrault it's not a syntax error; it's new syntax in ES2017. The airbnb config assumes you're transpiling with https://npmjs.com/babel-preset-airbnb which includes trailing function commas.
